packages/embed/src/index.ts

```diff
@@ -114,7 +114,8 @@ export function setup(setupConfig: SetupConfig): EmbedInstance {
 
   createPopupController(
     mutableRef<{ popup: Window; stopCallback: () => void }>(),
-    subjectManager
+    subjectManager,
+    config.popupTimeout ? config.popupTimeout * 60 : null
```

Comment:
Shouldn't this be converted to milliseconds (`config.popupTimeout * 1000`)?

Reply:
🤦🏻‍♂️ Thanks!